public/apps/configuration/panels/tenant-list/tenant-list.tsx

```diff
@@ -0,0 +1,371 @@
+/*
+ *   Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+import {
+  EuiBadge,
+  EuiButton,
+  EuiContextMenuItem,
+  EuiContextMenuPanel,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiInMemoryTable,
+  EuiLink,
+  EuiPageBody,
+  EuiPageContent,
+  EuiPageContentHeader,
+  EuiPageContentHeaderSection,
+  EuiPageHeader,
+  EuiPopover,
+  EuiText,
+  EuiTitle,
+  EuiGlobalToastList,
+  EuiOverlayMask,
+  EuiConfirmModal,
+} from '@elastic/eui';
+import React, { ReactNode, useEffect, useState, useCallback } from 'react';
+import { difference } from 'lodash';
+import { getAuthInfo } from '../../../../utils/auth-info-utils';
+import { AppDependencies } from '../../../types';
+import { Action, Tenant } from '../../types';
+import { renderCustomization } from '../../utils/display-utils';
+import {
+  fetchTenants,
+  transformTenantData,
+  fetchCurrentTenant,
+  resolveTenantName,
+  updateTenant,
+  requestDeleteTenant,
+  selectTenant,
+} from '../../utils/tenant-utils';
+import { getNavLinkById } from '../../../../services/chrome_wrapper';
+import { TenantEditModal } from './edit-modal';
+import { useToastState } from '../../utils/toast-utils';
+
+const PAGEID_DICT: { [key: string]: string } = {
+  DashboardId: 'dashboards',
```

Comment:
please use lower case for first character

Reply:
corrected